qiskit_machine_learning/connectors/torch_connector.py

```diff
@@ -218,13 +277,29 @@ def __init__(
                 output is only returned if the underlying neural network also returns sparse output,
                 otherwise it will be dense independent of the setting. Also note that PyTorch
                 currently does not support sparse back propagation, i.e., if sparse is set to True,
-                the backward pass of this module will return None.
+                the backward pass of this module will return None. Sparse support works on python
+                3.8 or higher.
+
+        Raises:
+            QiskitMachineLearningError: If the connector is configured as sparse and the underlying
+                network is not sparse. Or if python version is 3.7.
         """
         super().__init__()
         self._neural_network = neural_network
+        if sparse is None:
+            sparse = False
```

Comment:
In looking at `sparse` it says this
>If sparse is set to None, then the setting from the given neural network is used.

This here is forcing it to False though. Was wondering why it defaulted to None in init and then if None gets set to False here that the default would not just be False rather than None.

This adopting the sparse setting from the network seems no longer applicable as per the exception below

>"TorchConnector configured as sparse, the network must be sparse as well"

Reply:
Honestly, I missed the docstring. Updated `sparse` handling and the docstring. Should be better now.